<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrafficControl</class>
 <widget class="QMainWindow" name="TrafficControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trafficControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Sandbox</string>
     </attribute>
     <widget class="QTableView" name="trainListTableView">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>551</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="listTrainButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>94</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>List Trains</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>457</x>
        <y>160</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="listStationButton">
           <property name="text">
            <string>List Stations</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="stationListTableView">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>421</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="listTrackButton">
           <property name="text">
            <string>ListTracks</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="trackListTableView">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSpinBox" name="tickIntervalSpinBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="suffix">
       <string> ms</string>
      </property>
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="value">
       <number>500</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="runThreadCheckBox">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>11</y>
        <width>113</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Continue with ticks</string>
      </property>
     </widget>
     <widget class="QLabel" name="threadStatusButton">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>34</y>
        <width>38</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Interval</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Trains</string>
     </attribute>
     <widget class="QPushButton" name="createTrainButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>94</width>
        <height>33</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Train</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="createTrainCapacitySpinBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>131</width>
        <height>33</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="suffix">
       <string> passengers</string>
      </property>
      <property name="maximum">
       <number>600</number>
      </property>
      <property name="value">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Capacity</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="createTrainNameLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>41</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ID: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>211</height>
        </rect>
       </property>
       <property name="title">
        <string>Speed of train</string>
       </property>
       <widget class="QSpinBox" name="speedSpinBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>91</width>
          <height>33</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix">
         <string> km/h</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>-180</number>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
       </widget>
       <widget class="QLCDNumber" name="speedlcdNumber">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>72</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="emergencyBrakePushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>171</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Emergency Brake</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Desired Speed</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>80</y>
          <width>41</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>km/h</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Actual Speed</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Stations</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tracks</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QSpinBox" name="stepTimeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>49</width>
      <height>33</height>
     </rect>
    </property>
    <property name="suffix">
     <string> s</string>
    </property>
    <property name="maximum">
     <number>3600</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="timeTickButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>101</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Step time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateNetworkButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>510</y>
      <width>201</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate simple network</string>
    </property>
   </widget>
   <widget class="QPushButton" name="importNetworkButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>171</width>
      <height>33</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Lund_suburbs</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTrafficControl">
    <property name="title">
     <string>TrafficControl</string>
    </property>
   </widget>
   <addaction name="menuTrafficControl"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
